sources/ec.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+      "url": "http://www.aqvisor.net:9090/openaq/dataset",
+      "adapter": "quito",
+      "name": "REMMAQ",
+      "city": "Quito",
+      "country": "EC",
+      "description": "",
+      "sourceURL": "http://www.aqvisor.net:9090/openaq/location",
```

Comment:
sourceURL should be a link to the data source entity, so in this case: `http://www.quitoambiente.gob.ec/ambiente/`